clappr/clappr-ios
# Information for Metrics

## Goal
- Make explicit some info to use on metrics. Add `position` on seek events (`willSeek` and `didSeek`) and captures error from `playback`

## How to Test

1. Run unit tests
2. Add  the code bellow on `ViewController.swift#39`
3. Run project
4. Perform a seek and see the prints

```
player.on(Event.willSeek) { userInfo in
    if let position = userInfo?["position"] as? Double {
        print("on willSeek\nposition: \(position)")
    }
}

player.on(Event.didSeek) { userInfo in
    if let position = userInfo?["position"] as? Double {
        print("on didSeek\nposition: \(position)")
    }
}
```
